### PR TITLE
Update bio and affiliations

### PR DESCRIPTION
**Pull request by justaugustus**

### CODE REVIEW

**Comment by carlosonunez** [comment]
LGTM 👌🏾

File: content/_index.md [hunk 1]
```diff
@@ -3,4 +3,36 @@ title = "Stephen Augustus"
...
+conference.
+
+In 2020, Stephen also co-founded the [Inclusive Naming Initiative][ini], a
+cross-industry group dedicated to helping projects and companies make
+consistent, responsible choices to remove harmful language across codebases,
+standards, and documentation. He leads multiple workstreams here and maintains
+the initiative's infrastructure.
+
+In 2021, he joined Cisco as their first Head of Open Source within the Emerging
+Technologies & Incubation division.
```
Comment (celestehorgan) [inline content/_index.md]: All of these paragraphs start the same way (In [date]...) ; it's repetitive. Try:

```suggestion
Stephen was one of the Program Chairs for
[KubeCon / CloudNativeCon][kubecon], the cloud native community's flagship
conference, in 2020 and 2021. 

Stephen also co-founded the [Inclusive Naming Initiative][ini] in 2020, a
cross-industry group dedicated to helping projects and companies make
consistent, responsible choices to remove harmful language across codebases,
standards, and documentation. He leads multiple workstreams here and maintains
the initiative's infrastructure.

He is currently at Cisco, as their first Head of Open Source for the Emerging
Technologies & Incubation division. 
```

File: content/_index.md [hunk 2]
```diff
@@ -3,4 +3,36 @@ title = "Stephen Augustus"
 tagline = "Caesar of Systems"
 +++
 
-Friendly neighborhood Kubernetes guy at VMware. Previously Red Hat and CoreOS. Proud geek. Black engineer. Made in NYC.
\ No newline at end of file
+Stephen is a Black engineering director and leader in open source communities.
```
Comment (celestehorgan) [inline content/_index.md:6]:
```suggestion
Stephen is a Black engineering director and leader in open source communities. He focuses on building emergent cloud native technology and building inclusive open source communities.
```

**Comment by danpop-chainguard** [comment]
/lgtm 
well done.

File: content/_index.md [hunk 3]
```diff
@@ -3,4 +3,36 @@ title = "Stephen Augustus"
...
+
+He currently serves the Kubernetes community as a Chair for
+[SIG Release][sig-release], a Lead for [WG Naming][wg-naming], and an owner of
+[Enhancements][enhancements] subproject.
+
```
Comment (nicolefv) [inline content/_index.md:10]: possible alt: Additionally, he is a [CNCF SIG Contributor Strategy][contrib-strat] Chair and a [Dex][dex] maintainer.

Comment (justaugustus) [reply content/_index.md:10]: Thanks Nicole!
Integrated in 64967387b5640340cc499cd97273717f9e0fffc4.

**Comment by justaugustus** [comment]
Thanks everyone for the reviews and support! 💕

**Pull request merged**